package.json

```diff
@@ -48,6 +48,7 @@
   },
   "dependencies": {
     "@tailwindcss/vite": "^4.0.9",
+    "@tanstack/react-query": "^5.69.0",
```

Comment:
Was this an accident? I don't think this needs to be a dependency for the server. I could be wrong 🤷‍♀️

Reply:
yup, i will remove in the next PR with more loaders. Great catch!